configs/keycloak_bridge.yml

```diff
@@ -62,19 +70,17 @@ trustid-groups:
   - "end_user"
 
 # Keycloak configs
-keycloak-api-uri: http://localhost:8080
+keycloak-api-uri: http://keycloak.internal.local:8080
 keycloak-oidc-uri-map:
-  default: http://localhost:8080
-  trustid: http://idp.trustid.ch:8080
-  cloudtrust: http://idp.cloudtrust.io:8080
+  default: http://keycloak.local:8080
 keycloak-timeout: 5s
 
 # DB Audit RW
 # db-audit-rw-enabled was previously named events-db
 db-audit-rw-enabled: false
-db-audit-rw-host-port: 127.0.0.1:3306
-db-audit-rw-username: bridge
-db-audit-rw-password: bridge-password
+db-audit-rw-host-port: localhost:3306
```

Comment:
We must use host.docker.internal to be compatible with docker container in dev env